.buildkite/admin.yml

```
@@ -0,0 +1,17 @@
+agents:
+  queue: "cardano-wallet"
+
+env:
+  LC_ALL: "C.UTF-8"
+  NIX_PATH: "channel:nixos-21.11"
+  NODE_STATE_DIR: "/var/lib/buildkite-agent/cache/node/preprod"
+
+steps:
+  - block: "Refresh preprod node state"
+    key: refresh-preprod-node-state-block
+  - label: 'Refresh preprod node state'
+    depends_on: refresh-preprod-node-state-block
+    command:
+      - ./scripts/buildkite/release/fetch-preprod-snapshot.sh
```

```suggestion
    command:
      - ./scripts/buildkite/release/fetch-preprod-snapshot.sh
    env:
      NODE_STATE_DIR: "${STATE_DIR?}/node/preprod"
```

The `NODE_STATE_DIR` variable is essentially an argument that is specific to the `fetch-preprod-snapshot.sh` script. I think it's slightly more foolproof to set the argument here, in the immediate vicinity of the call of the script.

The `?` is explained here: https://buildkite.com/docs/agent/v3/cli-pipeline#environment-variable-substitution-requiring-environment-variables